tests/TestCase/View/Form/EntityContextTest.php

```diff
@@ -395,7 +395,10 @@ public function testSchemaOnCollections($collection)
         $this->assertSame('string', $context->type('99.title'));
         $this->assertNull($context->type('0.nope'));
 
-        $expected = ['length' => 255, 'precision' => null];
+        $expected = [
+            'length' => 255, 'precision' => null, 'type' => 'string',
+            'null' => null, 'default' => null, 'comment' => null, 'collate' => null,
```

Comment:
I don't see how the collate, type, or null keys would ever be useful in a form building context.

Reply:
I can see how null could be useful for indicating fields are optional. Would you be ok with only exposing null and comment? We have existing APIs to get an abstract type for a field.

Reply:
I have updated `EnityContext` to addtionally expose only `null`, `comment` and `default`.